models/lioncore.json

```diff
@@ -0,0 +1,661 @@
+[
```

Comment:
Is this the serialization of the LIonCore's self definition? If so, why do we need to commit it?

Reply:
Same as above: to see when it changes.